config.toml

```diff
@@ -18,6 +18,17 @@ enableRobotsTXT = true
   copyrightMessage = '© SQUARE ENIX CO., LTD. All Rights Reserved | FINAL FANTASY is a registered trademark of Square Enix Holdings Co., Ltd. | All content © their respective authors | The Balance is a non-profit community-owned website.'
 
 [markup]
+  [markup.goldmark]
+    [markup.goldmark.renderer]
```

Comment:
This was simply moved from below in the file. It's not to my knowledge necessary for this functionality.